bundles/AdminBundle/Resources/public/js/pimcore/object/tags/classificationstore.js

```diff
@@ -416,14 +416,15 @@ pimcore.object.tags.classificationstore = Class.create(pimcore.object.tags.abstr
                 context["applyDefaults"] = true;
             }
 
+            context["type"] = this.type;
```

Comment:
what I was suggestion was to manipulate the tooltip here instead of the central edit.js (which shouldn't be even aware of the data type it is dealing with). what do you think ?

Reply:
Well, I honestly don't know how I would best solve that.

At this point I am in the stroe context, not in the field.

I would have to run over the elements and change the text there somehow and in the recursive layout method I would still have to check if the text is already translated because otherwise the translated text would be translated.

Reply:
well, childItem is already the thing you need.

But I guess special handling would be still needed in edit.js. If you remove the if (...classificationstore ...) then the tooltip wil be still retranslated. So let's leave it as it is for now.